srs_data/tagging.py

```diff
@@ -28,7 +32,16 @@ def input_file(self):
             f'{CLIENT_FULL_NAMES[self._client]}-tagsource{self._start_date}.csv'
         )
 
+    @property
+    def uniform_sample_file(self):
+        return self._autotagged_uniform_sample_file
+
+    @property
+    def uniform_sample_file_updated_with_client_tags(self):
+        return self.uniform_sample_file.replace('.csv', '_auto.csv')
+
     def _run_steps(self):
+        self._update_the_original_uniform_sample()
```

Comment:
Also commented in the other pull request (didn't realize they were overlapping). But this should be split up into pulling the original sample file, and then updating that file instead of assuming the file is somewhere on your computer. This also makes sure you're 100% sure you're updating the right file

Reply:
Great point.